### ISSUE TITLE
Nextcloud Feature Request- JavaScript XMPP Chat by group 

### ISSUE THREAD

**Issue by greavette** [jsxc/jsxc#636]
Hello Team,

Great App here...very useful and I thank you for making this available for all to use.

I use JavaScript XMPP Chat within my Nextcloud 12.0.2.  One feature I would like to propose is being able to provide this chat app by group.  

We have a use case whereby I've enabled another App called External Sites.  We have some users of our Nextcloud who will only use the External Sites App we provide them and they will not use the chat feature provided by JavaScript XMPP Chat.

Having Nextcloud JavaScript XMPP Chat give us the ability to provide chat functionality by group would be very useful for us.

Thank you.

**Comment by greavette** [comment]
Additional comments on this feature request.

I've just noticed that there jxsc cloud repo within GitHub.  Have I put my feature request in the right location?

Secondly...I found the following feature request that I think may be related to what I'm asking.  Could someone confirm if my request to allow the Nextcloud Chat App by group is already being worked on in the following request?

https://github.com/nextcloud/jsxc.nextcloud/pull/57

Thank you.

**Comment by sualko** [comment]
> Have I put my feature request in the right location?

Yes you did. I the past I tried to collect all issues in this core issue tracker, but as we moved the jsxc.nextcloud repo to the nextcloud organisation the issue tracker got enabled there as well.

> Could someone confirm if my request to allow the Nextcloud Chat App by group is already being worked on in the following request?

The pr is about the internal server, so no, nobody is currently working on this enhancement, but if you like you can [add a bounty](https://www.bountysource.com/issues/50710610-nextcloud-feature-request-javascript-xmpp-chat-by-group) to this issue to motivate developers, or you can open a pr by yourself. I really would appreciate the second option :wink:.
